client/components/UseStyles.js

```diff
@@ -0,0 +1,103 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
We should consider the `styled-components` alternative as well as `material-ui@5`

https://github.com/mui-org/material-ui/issues/20012

Reply:
sounds good. So styled `styled-components` allows you to override/modify material-ui components right?